### PR TITLE
Houssame working branch

### PR DESCRIPTION
**Pull request by HoussameH**
Formatting, documentation, and update to Calc_TS with function for merra and raster Data loading and calculation

### CODE REVIEW

File: code/Master.py [hunk 1]
```diff
@@ -731,13 +745,29 @@ def generate_buffered_population(paths, param):
...
 
-def generate_wind_correction(paths, param):
+def generate_area(paths, param):
+    timecheck('Start')
+    Crd_all = param["Crd_all"]
+    n_high = param["n_high"]
+    res_desired = param["res_desired"]
+
+    # Calculate available Area
+    A_area = calc_areas(Crd_all, n_high, res_desired)
```
Comment (kais-siala) [inline code/Master.py]: I was thinking to completely move the content of "calc_areas" into "generate_area"

File: code/Master.py [hunk 2]
```diff
@@ -1152,22 +1135,22 @@ def reporting(paths, param, tech):
...
             if regions.loc[ind_prev, "Available_Masked"] > int(available_masked):
                 regions.drop([reg], axis=0, inplace=True)
                 continue
             else:
-                regions.drop([ind_prev], axis=0, inplace=True)  
-        
-        # Sum area: available area in km2
+                regions.drop([ind_prev], axis=0, inplace=True)
+
+                # Sum area: available area in km2
```
Comment (kais-siala) [inline code/Master.py]: no indent

File: code/config.py [hunk 3]
```diff
@@ -101,24 +104,24 @@ def scope_paths_and_parameters(paths, param):
...
-  
-    
+
+
 def computation_parameters(param):
     """
     This function defines parameters related to the processing.
     Some modules in ``Master.py`` allow parallel processing. The key *nproc*, which takes an integer as a value, limits the number of parallel processes.
-    *CPU_limit* is a boolean parameter that ?????
+    *CPU_limit* is a boolean parameter that sets the level of priority for all processes in the multiprocessesing, leave True if you plan on using the computer while FLH and TS are being computed, False for fastest computation time
```
Comment (kais-siala) [inline code/config.py]: ``True``
``False``

File: code/config.py [hunk 4]
```diff
@@ -251,21 +260,25 @@ def landuse_parameters(param):
...
+
     landuse = {"type": np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]),
-            "type_urban": 13,
-            "Ross_coeff": np.array(
-                [0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208,
+               "type_urban": 13,
+               "Ross_coeff": np.array(
+                   [0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208, 0.0208,
+                    0.0208,
                     0.0208, 0.0208, 0.0208, 0.0208]),
```
Comment (kais-siala) [inline code/config.py]: this is ugly

File: code/Master.py [hunk 5]
```diff
@@ -1595,7 +1633,6 @@ def obj_expression(model):
 
         else:
             r = np.full((len(param["quantiles"]), len(settings)), 0)
-            no_sol_low_high = no_sol_low_high + reg + ', '
```
Comment (kais-siala) [inline code/Master.py]: why?

Comment (HoussameH) [reply code/Master.py]: I removed it cause it's not possible for a region to be both too high and too low

**Pull request merged**